recipes/qt/6.x.x/conanfile.py

```diff
@@ -616,6 +616,13 @@ def _configure_cmake(self):
         self._cmake.definitions["FEATURE_pkg_config"] = "ON"
         if self.settings.compiler == "gcc" and self.settings.build_type == "Debug" and not self.options.shared:
             self._cmake.definitions["BUILD_WITH_PCH"]= "OFF" # disabling PCH to save disk space
+            
+        if tools.cross_building(self):
```

Comment:
@ericLemanissier 
I'm in no way a qt specialist.
I remember reading to cross build qt for android, you first need a native qt.
Would the following be needed instead? + setting the correct appropriate arguments.
```python
def build_requirements(self):
    if tools.cross_building(self):
        self.build_requires(f"{self.name}/{self.version}")
```

Reply:
that's the way qt 6 handles cross compilation yes, but I don't think conan allows it.

Reply:
You mean conan does not generate the required cmake modules for build requirements?
Is there missing something else?
Imho, this deserves to be added to conan.

Reply:
the thing is that you usually want different options for the `build` qt (basically just the minimum) from the `host` qt.